logging/troubleshooting/cluster-logging-must-gather.adoc

```diff
@@ -0,0 +1,26 @@
+[id="cluster-logging-must-gather"]
+= Collecting logging data for Red Hat Support
+include::modules/ossm-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+
+
+toc::[]
+
+When opening a support case, it is often helpful to provide debugging information about your cluster to Red Hat Support.
```

Comment:
s/often//

Reply:
@adellape I got this line from the _About the must-gather tool_ module. Worth changing there, too?